src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/MuTect2.java

```diff
@@ -1041,5 +1041,8 @@ public ActivityProfileState isActive(final AlignmentContext context, final Refer
             return new ActivityProfileState( ref.getInterval(), prob, ActivityProfileState.Type.NONE, null);
         }
     }
+
+    public static boolean hasBadMate(final GATKRead rec) {
```

Comment:
We should document (and perhaps rename); GATK3 has the following docs about it: "If mates in a pair are mapping to different contigs, it is likely that at least one of them is in the wrong place."